modules/user_manual/pages/apps/admin_audit.adoc

```diff
@@ -0,0 +1,198 @@
+= The Admin Audit App
+:toc: right
+:toclevels: 1
+:leveloffset: +1
+
+== Introduction
+
+The Admin Audit app is an auditing module for ownCloud to trace the actions of users and administrators.
+
+== Configuration
+
+Configuration is needed to redirect the messages into a separate file.
+
+Add these lines to "config.php" and adjust the target path accordingly:
+
+[source,console]
+----
+'log.conditions' => [
+  [
+    'apps' => ['admin_audit'],
+    'logfile' => '/var/www/owncloud/data/admin_audit.log'
+  ]
+]
+----
+
+Please note that the target path must be writeable for the web server user.
+All messages regardless of log level will be logged there.
+To ignore all CLI triggered events (not the default), you can set the following option:
+
+[source,console]
+----
+occ config:app:set admin_audit ignore_cli_events --value='yes'
```

Comment:
nope. use the predefined macros (attributes+ ect)

Reply:
Yep, as that is a direct copy of the project's README file.